--- Mothur.source/makefile	2013-06-13 14:47:25.000000000 +0100
+++ Mothur.source.1.31.2/makefile	2013-06-28 10:20:58.122668317 +0100
@@ -13,7 +13,7 @@
 64BIT_VERSION ?= yes
 USEREADLINE ?= yes
 CYGWIN_BUILD ?= no
-USECOMPRESSION ?= no
+USECOMPRESSION ?= yes
 MOTHUR_FILES="\"Enter_your_default_path_here\""
 RELEASE_DATE = "\"6/13/2013\""
 VERSION = "\"1.31.2\""
@@ -28,7 +28,7 @@
 	#CXX = g++44
 	
 	#if you are a mac user use the following line
-	TARGET_ARCH += -arch x86_64
+	#TARGET_ARCH += -arch x86_64
 	
 	#if you using cygwin to build Windows the following line
 	#CXX = x86_64-w64-mingw32-g++
@@ -37,7 +37,7 @@
  	#TARGET_ARCH += -m64 -static
 
 	#if you are a linux user use the following line
-	#CXXFLAGS += -mtune=native -march=native -m64
+	CXXFLAGS += -mtune=native -march=native -m64
 	
 	CXXFLAGS += -DBIT_VERSION 
     FORTRAN_FLAGS = -m64
