--- Mothur.source/makefile	2012-08-06 14:15:32.000000000 +0100
+++ Mothur.source.new/makefile	2013-02-20 15:05:33.146621001 +0000
@@ -13,7 +13,7 @@
 64BIT_VERSION ?= yes
 USEREADLINE ?= yes
 CYGWIN_BUILD ?= no
-USECOMPRESSION ?= no
+USECOMPRESSION ?= yes
 MOTHUR_FILES="\"Enter_your_default_path_here\""
 RELEASE_DATE = "\"8/8/2012\""
 VERSION = "\"1.27.0\""
@@ -28,7 +28,7 @@
 	#CXX = g++44
 	
 	#if you are a mac user use the following line
-	TARGET_ARCH += -arch x86_64
+	#TARGET_ARCH += -arch x86_64
 	
 	#if you using cygwin to build Windows the following line
 	#CXX = x86_64-w64-mingw32-g++
@@ -37,7 +37,7 @@
  	#TARGET_ARCH += -m64 -static
 
 	#if you are a linux user use the following line
-	#CXXFLAGS += -mtune=native -march=native -m64
+	CXXFLAGS += -mtune=native -march=native -m64
 	
 	CXXFLAGS += -DBIT_VERSION 
     	FORTRAN_FLAGS = -m64
