--- Mothur.source/uchime_src/mk	2013-02-15 15:27:03.000000000 +0000
+++ Mothur.source.1.31.2/uchime_src/mk	2013-06-28 10:21:08.460710133 +0100
@@ -14,9 +14,9 @@
 done
 
 LINK_OPTS= 
-if [ `uname -s` == Linux ] ; then
-    LINK_OPTS=-static
-fi
+#if [ `uname -s` == Linux ] ; then
+#    LINK_OPTS=-static
+#fi
 g++ $LINK_OPTS $ENV_LINK_OPTS -g -o uchime $ObjNames >> mk.stdout 2>> tmp.stderr
 cat tmp.stderr
 cat tmp.stderr >> mk.stderr
